pkgs/os-specific/linux/hdparm/default.nix

```diff
@@ -1,12 +1,12 @@
 { stdenv, fetchurl }:
 
 stdenv.mkDerivation rec {
-  name = "hdparm-9.58";
+  pname = "hdparm";
+  version = "9.58";
 
   src = fetchurl {
-    url = "mirror://sourceforge/hdparm/${name}.tar.gz";
+    url = "mirror://sourceforge/hdparm/hdparm-${version}.tar.gz";
```

Comment:
```suggestion
    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
```

Reply:
if this is to remain stable in the future, I think ti's fine as is